server/controllers/finance/debtors/groups/index.js

```diff
@@ -276,9 +276,11 @@ function invoices(req, res, next) {
  */
 function loadInvoices(params) {
 
+  const { debtorByuuid } = params;
+  const key = debtorByuuid ? 'uuid' : 'group_uuid';
```

Comment:
If you use the `DebtorService`, you won't need to do this and we won't need to write any more integration tests for this functionality.